### DISCUSSION TITLE
when can we expect support for mongo-server 5.1 with mongo-java-driver upgrade?

### DISCUSSION THREAD

**Discussion by RajanAhlawatSpire** [MorphiaOrg/morphia#2107] [Q&A]
We are planning to incorporate morphia, but latest version is still using java driver 4.2.3 which supports till monog-server 4.4. By when we can expect the upgrade and support for 5.1 mongo server

**Comment by evanchooly** [comment]
Morphia should work fine against 5.x and even 6.x.  You can see the test matrix [here](https://github.com/MorphiaOrg/morphia/actions/runs/3109133776).  2.3 should be out in the next couple of weeks bringing everything up to the latest ... everything. I've been traveling for conference work and taking care of some other things but I'm hoping to get it done soon.
